sass/components/VmDetails/vm-details.scss

```diff
@@ -39,6 +39,11 @@
   width: 400px;
   flex: 1;
   flex-basis: auto;
+  margin-right: 8px;
```

Comment:
hmm, I guess we could also add this to `kubevirt-vm-details__details-list` just to be sure?

Reply:
meaning the word-break, not sure about the margins..

Reply:
There is already margin-right:20px; there which I think is ok. I just added a minimal amount just in case we have this long string for the name.

Reply:
Ok. Can we have a  `word-break` there?

Reply:
oh sorry didn't see your second comment :)